### ISSUE TITLE
[PROJECT] New Model for Platform Fees

### ISSUE THREAD

**Issue by piamancini** [opencollective/opencollective#3592]
## Purpose
(Why have we prioritised this?)
We've been experimenting with a host plan all year long and we are unhappy with it. We've been charging hosts based on number of collectives and that is stressing them out since they want to give people a chance to raise funds first. It is also confusing and adding complexity to our proposition. 

Related Issue - Better Plans for Hosts
https://github.com/opencollective/opencollective/issues/3393

[Google Doc]( https://docs.google.com/document/d/1rAEXGaFtJYj2kPIE_bKBc5Cq-ieB6Bjrp0eIF38E6mo/edit#heading=h.m8hds7vwfcdo)

We want to build a simple proposal for third party hosts that makes sense for them and enables us to focus on the platform + first party host proposal that has proven to have better market fit. 

## Scope

TL;DR
the Platform is free for Collectives
Platform Tips (aka “Fees on Top”) is the new default (but not for OSC)
0% Host Fee is the default for new Hosts
Collectives only have a financial relationship with their Hosts (but can still be self hosted)
Platform only have a financial relationship with Hosts
Hosts can charge fees for Payments and also for Payouts (new)
Platform should not charge for money it’s not processing (Added Funds, Bank Transfers, Manual Expenses)
Platform invoice Hosts (and self hosted Collectives) monthly for the money due

## Implementation

The general rule is to drop mandatory Platform Fees for all Hosts and to switch to Platform Tips. Also, all new Hosts will have 0% Host Fee and will not be able to set Host Fees unless they subscribe for an Host Plan.

The new default Free Tier is: 0% Host Fee, 0% Platform Fee. Platform Tips.

Like today Host fees will be configurable globally at the Host level and specifically for each Collective. Host should be able to set:
Fees on Contributions
Credit Card (config)
PayPal (config)
Bank Transfers (config)
Added Funds (ad hoc, per transaction)
Fees on Expenses
PayPal (config)
Transferwise (config)
Manual Payments (ad hoc, per transaction)

It’s the Host responsibility to wire the money collected on behalf of the Platform at the end of the month. Other costs and fees can be added to this invoice if necessary. This will be automated by using our Expense and invoicing system.
The new default

Host Fees are defined by the Host  0 - whatever
Discard Host Plans
15% on the monthly Host Revenues, accounted on the monthly invoice 
0% Platform Fee on Contributions
Platform Tips (“Fees On Top”)
The Host will be invoiced by the platform monthly for money due
Uncollected Platform Tips (PayPal, Bank Transfers)
Revenue Share program (OSC, Foundation)

### Exceptions (1st Party Hosts)
Out of the default, we will make fees customizable so we can implement the following exceptions.
Open Source Collective exception
The OSC will move to 10% Host Fee, 0% Platform Fee (for Collectives)
No Platform Tips (aka “Fees On Top”)
No fees for Payouts
The OSC will be invoiced by the Platform monthly:
50% of the collected Host Fees
Let donors cover the overhead. (would you like to cover the fees this collective pays?)
Open Collective Foundation exception

The Foundation will move to 5% - 8% Host Fee, 0% Platform Fee (for Collectives)
5% for Credit Card and PayPal (customizable per Collective)
8% for Added Funds and Bank Transfers (customizable per Collective)
Platform Tips (“Fees On Top”)
No fees for Payouts
The OSC will be invoiced by the platform monthly:
50% of the collected Host Fees

### Other exceptions
We plan to have custom arrangements for Hosts over > $100.000 in yearly money processed.

## Owner
@znarf and @piamancini 

*Note: Project owner responsibilities include coordinating all the different people that need to contribute to the project, defining specs and goals, communicating project progress, being the point person for decisions, QA, documentation, and a plan for ongoing maintenance. If the project owner can't continue, they need to hand the role over to another person explicitly.*

## Prioritization

- Does this serve our mission?

Yes - it is a better proposition for small and large hosts as well as for the platform. It will free hots to grow without paying anything is this is not a business for them and it aligns our interests much more with them when they grow their revenue. 

- Will this help to make us financially sustainable within a year?

Yes - it's designed to support more small hosts and build strategic partnerships with the bigger ones. 

- Is it feasible to build and maintain with our team and runway?

Yes 

- Can this make us move faster?

It will reduce the complexity in general, reduce time spent on support. 

- Does this increase quality and not technical debt?

yes - we are getting rid of complicated host plans, we are improving and automating how we handle non stripe platform fees. 

- Will this bring us more Collective and financial contributors?

Yes - by letting hosts add as many collectives as they want. 

- Is this already solved elsewhere or is it our unique contribution?

It's part of the fiscal sponsorship as a service + platform offer that is unique to us. 

## Roadmap & Timeframe
(Key milestones or phases and estimated delivery dates, including specs, design, feature development, feedback, marketing/comms, and documentation)

### Tasks

- [ ] Monthly Platform Invoicing for Hosts
- [ ] Collected Platform Tips
- [ ] Revenue Share (ie: 15%, 50%)
- [ ] Make 0% Platform Fees and 0% Host Fees the defaults
- [ ] Make “Host with Budget” the default
- [ ] API helpers to compute key metrics
- [ ] Amount in processed Contributions (Credit Card, PayPal)
- [ ] Amount in processed Expenses (PayPal, Transferwise)
- [ ] Host Revenues
- [ ] Pricing pages
- [ ] Update the Platform one
- [ ] New pricing section/page for each Host (based on the Host Settings)
- [ ] Revamp Self Hosting so Collectives can directly connect Stripe / Bank Accounts
- [ ] Prevent USERs becoming Fiscal Hosts
- [ ] Ability to set Host Fees
- [ ] Ability to set global Host Fees in Host Settings (per Payment method)
- [ ] Ability to set granular Host Fees on a Collective basis (per Payout method)
- [ ] Disclaimer that the Platform will take 15% of it
- [ ] Discard Host Plans
- [ ] Design: Hosts creation and Hosts onboarding

Other Tasks 
- [ ] Pricing Page 
- [ ] Deprecate single collective + own host in favor of collective with payments connected. 
- [ ] Deprecate user hosting in favor of collective with payments connected.

### Migration

To ease the transition, we should consider making the transition manual for the Hosts. So that they can properly opt in and understand the changes.

Everyone should be moved to the new scheme by January 2022.

## Success metrics
(What we will judge outcomes of the project by)

**Comment by alanna** [comment]
@znarf - @piamancini said I should check in with you about the onboarding plan for the new platform fees for hosts. 

I'd like to start manually putting some new hosts I'm working with on it so we can test it, and manually handle the invoicing or whatever needs to happen for MVP. For example, the [new NZ charity host](https://opencollective.com/giftcollective) will be going live soon and the admin is keen to be on the new 15% plan.

Is this a good system for MVP:
* Set up host 
* platform tips/fees on top turned on
* host budget turned on
* request you to put them on some kind of setting that doesn't require them to get on a paid host plan even when they hit the threshold, but all the paid features are turned on
* I manually submit an expense to the host budget each month for 15% of their host fees plus any platform tips given outside Stripe (I may need your help to confirm these numbers)
* We keep a spreadsheet of the hosts on this MVP setting so we can keep track of them and make sure we're doing the invoicing

Migration plan:
I'd also like to help with some preliminary work to communicate to hosts and help get us ready for switching on the new 15% plan when we're ready.
* All hosts will need their host budgets turned on so we can automate creating expenses for platform fees in the future, so should we ask them to switch it on or actually just force it on for all of them now, and if so we'll need to communicate that.
* Offer the option to switch to the new plan now, in the MVP process explained above
* What else needs to happen now so we'll be ready for the new plan down the track?

**Comment by piamancini** [comment]
MVP looks doable to me.

**Comment by znarf** [comment]
@alanna We'll have all the bricks for the MVP soon. I'm not sure about migration details yet.

We should definitely continue having Hosts activate their budget feature and see if there is any blocker before activating it for everyone. I can see problems and questions around the balance (as it's new to them and can be negative when we activate it).

**Comment by alanna** [comment]
OK I will proceed with putting new hosts on the MVP version of the new plan if they want to opt in. When a host wants to activate it, I'll ask @znarf to put them on an unlimited free host plan with fees on top, and I'll manage the manual invoicing for platform fees.

@znarf can you please export data of hosts who have no activated their host budget, including the admin emails and names, and I'll prepare a communication to them asking them to activate it optionally for now. I'll let you know who wants to and you can try doing some and see what issues arise.

**Comment by alanna** [comment]
@znarf can you please activate the $0 host plan for [OCNZ](https://opencollective.com/ocnz) and [GiftCollective](https://opencollective.com/giftcollective) now? I'm tracking them on [this spreadsheet](https://docs.google.com/spreadsheets/d/1smZDyvJB3qKbjncrqcvrnlD-5YNXfnJGtGsQL0otJtM/edit#gid=0).

**Comment by alanna** [comment]
I would like to push for #3362 to be included in this project if possible. We want to give hosts the tools they need to autonomously figure out their business model with their Collectives. We can still charge our 15% regardless of if they charge a % or a flat rate fee.

**Comment by alanna** [comment]
I have been talking to several host admins about the new pricing model, and overall they are positive about it. We have a few test cases activating now to try it out.

I think it makes perfect sense for:

1. Hosts who charge a host fee through the platform on added funds
2. Host who charge 0% host fee, but have many Collectives doing crowdfunding and accepting contributions through the platform, where we can get platform tips

Where it doesn't make as much sense:

3. Hosts who mostly add funds manually and don't charge a host fee through the platform, which breaks down into: 
  3a. hosts who genuinely don't charge a fee
  3b. hosts who charge their fee outside the platform, i.e. just add 90% of a grant to Open Collective instead of 100% with a 10% fee)

These further break down into:

A. Hosts who feel OC adds value and they want to pay us something but we need to give them an easy way to do it
B. Hosts who will try to avoid paying OC even if they are getting value

When it comes to A, hosts have actually been pushing back on me saying OC should charge them something and it's not fair. So, I think a good idea is to add `platform tips` to the `add funds` box, so they can optionally allocate some funds to OC. Then we can include any amounts entered here on their monthly platform fees expense.

For B, we are relying on the "talk to us if you process over $50,000 or $100,000" caveat. In that case, we'll need to just come to some mutual agreement about platform fees, or in the worst case, threaten to kick them off I guess? I feel like we might be able to come up with a better solution here, but maybe that's good enough to go forward for one year and see how it works out.

**Comment by alanna** [comment]
Is this done? @piamancini 

Maybe only #3811 is outstanding?

**Comment by znarf** [comment]
I'm closing this. https://github.com/opencollective/opencollective/issues/3811 Will be tackled out of the project.

**Comment by BenJam** [comment]
> I'm closing this.

FTFY
